index.html

```diff
@@ -0,0 +1,186 @@
+                <li>Javascript</li>
+              </ul>
+              <div class="btn-project">
+               <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+              
+              </div> 
+            </div>
+
+
+              <div class="project5-img">
+              </div>
+              <div class="project-des">
+                <h2 class="sub-title">Project name goes here</h2>
+                <ul>
+                   <li>HTML/CSS</li>
+                   <li>Ruby on Rails</li>
+                   <li>Javascript</li>
+                </ul>
+                <div class="btn-project">
+                 <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+                </div> 
+              </div>
+
+                <div class="project6-img">
+                </div>
+                <div class="project-des">
+                  <h2 class="sub-title">Project name goes here</h2>
+                  <ul>
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>Javascript</li>
+                  </ul>
+                  <div class="btn-project">
+                   <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+                  </div> 
+                </div>
+          </div>
+ </section>
+
+ <section class="aboutme">
+    <h2 class="aboutme-title">
+       About
+    </h2>
+    <p class="aboutme-p">
+      I can help you build a product , feature or website Look through some of my work 
+      and experience! If you like what you see 
+      and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <button class="aboutme-btn">
+       <span> Get my  CV</span> <img src="./images/ic_download.png" alt="ic_download">
+    </button>
+    <div class="aboutme-cards">
+      <div class="aboutme-techs">
+        <h3 class="aboutme-techs-title">Languages</h3>
+        <ul class="techs">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+            <li>CSS</li>
+        </ul>
+        <img class= "logo-corner" src="./images/ic_languages_mob.png" alt="languages">
+      </div>
+      <div class="aboutme-framew">
+        <h3 class="aboutme-techs-title">Frameworks</h3>
+        <ul class="framew">
+          <li>React.js</li>
+          <li>Ruby on rails</li>
+          <li>RSpecs</li>
+          <li>Capibara</li>
+        </ul>
+        <img class= "logo-corner" src="./images/ic_framework.png" alt="Frameworks">
+      </div>
+      <div class="aboutme-skills">
+        <h3 class="aboutme-techs-title">Skills</h3>
+        <ul class="skills">
+          <li>Database Management</li>
+          <li>CLI</li>
+          <li>API Design</li>
+          <li>Version Control</li>
+          <li>Web Development</li>
+        </ul>
+        <img class= "logo-corner" src="./images/ic_skills.png" alt="skills">
+      </div>
+    </div>  
```

Comment:
- [ ] Kindly use ul and li tags instead of div tags for the About-me technology categories. They are semantic tags and it informs those who use screen readers that the section has list items. 👍 😎 

#### Hint

```HTML
<ul class="aboutme-cards">
  <li class="aboutme-techs">
    <h3 class="aboutme-techs-title">Languages</h3>
    <ul class="techs">
      <li>Javascript</li>
      <li>Ruby</li>
      <li>HTML</li>
        <li>CSS</li>
    </ul>
    <img class= "logo-corner" src="./images/ic_languages_mob.png" alt="languages">
  </li>
  <li class="aboutme-framew">
     ...
   </li>
  <li class="aboutme-skills">
    ...
  </li>
</ul>  
```